### PR TITLE
 Enforce PEP263 for PyTorch python codebase

### PR DESCRIPTION
**Pull request by malfet**
All python files containing non-ASCII characters should be correctly annotated with `# -*- coding: utf-8 -*-` comment

Delete number of superfluous UTF-8 characters, most commonly UTF-8 opening closing quotation mark U+2019 (’) instead of ascii apostrophe ', for example `Module’s`->`Module's`

### CODE REVIEW

**Review by albanD — COMMENTED** [review]
Looks like flake8 expects to see this coding magic everywhere or it throws a warning (which we turn into an error)

**Comment by malfet** [comment]
@albanD , for the time being forked the plugin to make it optional for files that do not have utf-8

File: requirements-flake8.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@ flake8==3.8.2
 flake8-bugbear==20.1.4
 flake8-comprehensions==3.3.0
 flake8-executable==2.0.4
+-e https+git://github.com/malfet/flake8-coding.git#egg=flake8-coding
```
Comment (samestep) [inline requirements-flake8.txt:5]: What's the reason for using this instead of the PyPI library?
```suggestion
flake8-coding==1.3.2
```

Comment (malfet) [reply requirements-flake8.txt]: Because flake8-coding does not allow skipping annotation for files that do not have utf-8 characters

Comment (samestep) [reply requirements-flake8.txt]: Gotcha, makes sense; I think at the time I posted the comment, you hadn't yet pushed https://github.com/malfet/flake8-coding/commit/1106d953dde998ab6dd7f951efc804dea562d609

**Comment by malfet** [comment]
@samestep I found a good example to add to trailing whitespace linter:
https://github.com/pytorch/pytorch/blob/da7a27b84750a3aa82a5837c361f8ae8ee8f7817/torch/onnx/__init__.py#L85
It ends with number of non-breakable whitespaces U+00A0

**Comment by samestep** [comment]
> @samestep I found a good example to add to trailing whitespace linter:
> https://github.com/pytorch/pytorch/blob/da7a27b84750a3aa82a5837c361f8ae8ee8f7817/torch/onnx/__init__.py#L85
> 
> 
> It ends with number of non-breakable whitespaces U+00A0

Interesting! I'm very curious how those got there 🤔 I'll see if I can modify the regex to catch that sort of thing

**Review by samestep — APPROVED** [review]

**Pull request closed without merging**